src/lib.rs

```diff
@@ -447,17 +445,27 @@ where
                     CONTENT_TYPE,
                     HeaderValue::from_static("text/plain; version=0.0.4; charset=utf-8"),
                 );
-                body = BoxBody::new(inner.metrics());
+
+                EitherBody::right(BoxBody::new(StreamLog {
```

Comment:
from the current code, it seems like you still want to wrap this other response type in StreamLog?